.prepare_deploy

```diff
@@ -22,29 +22,42 @@ This binary was compiled on $(date -u).
 $(git log -n 3)
 END
 
-for file in ./*.exe
-do
-  zip "${file%.*}.zip" README.txt LICENSE.txt "$file"
-done
-
-for file in *.linux-x86_64
-do
-  base="${file%.*}"
-  cp "$file" "shellcheck"
-  tar -cJf "$base.linux.x86_64.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
-  rm "shellcheck"
-done
-
-for file in *.linux-armv6hf
-do
-  base="${file%.*}"
-  cp "$file" "shellcheck"
-  tar -cJf "$base.linux.armv6hf.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
-  rm "shellcheck"
-done
+if [ "$TRAVIS_OS_NAME" = 'linux' ];
+then
+  for file in ./*.exe
+  do
+    zip "${file%.*}.zip" README.txt LICENSE.txt "$file"
+  done
+
+  for file in *.linux-x86_64
+  do
+    base="${file%.*}"
+    cp "$file" "shellcheck"
+    tar -cJf "$base.linux.x86_64.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
+    rm "shellcheck"
+  done
+
+  for file in *.linux-armv6hf
+  do
+    base="${file%.*}"
+    cp "$file" "shellcheck"
+    tar -cJf "$base.linux.armv6hf.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
+    rm "shellcheck"
+  done
+fi
+
+if [ "$TRAVIS_OS_NAME" = 'osx' ];
+then
+  for file in *.darwin-x86_64
+  do
+    base="${file%.*}"
+    cp "$file" "shellcheck"
+    tar -cJf "$base.darwin.x86_64.tar.xz" --transform="s:^:$base/:" README.txt LICENSE.txt shellcheck
```

Comment:
Will this `tar` command work on macOS?

Reply:
It won't. To handle `--transform` in OS X, we will need to install `gnu-tar`. It then becomes available as `gtar`.

Homebrew is available in the OS X Travis:
https://docs.travis-ci.com/user/reference/osx/#homebrew